src/components/form/datePicker.js

```diff
@@ -1,30 +1,114 @@
-import React from "react";
+import React, { forwardRef, useState } from "react";
 import PropTypes from "prop-types";
 import { ListHeader } from "../list-header/list-header";
+import ReactDatePicker from "react-datepicker";
 
-export const DatePicker = ({ question, onAnswer, currentResponse }) => {
-  const onChange = event => {
+import ro from "date-fns/locale/ro";
+
+import "react-datepicker/dist/react-datepicker.css";
+import "./datePicker.css";
+
+export const DatePicker = ({
+  withTime,
+  question,
+  onAnswer,
+  currentResponse
+}) => {
+  const [startDate, setStartDate] = useState(
+    currentResponse ? currentResponse : null
+  );
+
+  const onChange = date => {
+    setStartDate(date);
     const answer = {
       questionId: question.questionId,
-      value: event.target.value
+      value: date
     };
     onAnswer(answer);
   };
 
+  const getDatePicker = () => {
+    if (withTime) {
+      return <DateTimePicker startDate={startDate} onChange={onChange} />;
+    } else {
+      return <DateOnlyPicker startDate={startDate} onChange={onChange} />;
+    }
+  };
+
   return (
-    <div>
+    <div className={"date-picker"}>
       <ListHeader title={question.questionText} />
-      <p>Here is a custom component: </p>
-      <input onChange={onChange} value={currentResponse} type="date" />
+      {getDatePicker()}
     </div>
   );
 };
 
+const DateOnlyPicker = ({ startDate, onChange }) => {
+  return (
+    <ReactDatePicker
+      customInput={<CustomInput />}
+      placeholderText={"Introdu data"}
+      selected={startDate}
+      onChange={onChange}
+      locale={ro}
+      dateFormat={"d MMMM yyyy"}
```

Comment:
Is this a typo?

Reply:
`d`? No, the difference between `d` and `dd` in that context is that for days under 10 - let's say 2 - `d` outputs `2` while `dd` outputs `02`. For days greater than 10 there is no difference